nixer-spring-plugin-demo-app/e2e-tests/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "e2e-tests",
+  "version": "1.0.0",
+  "description": "This module contains e2e tests based on postman simulating variety of cases for user login.",
```

Comment:
I propose a description that would explain need for custom newman
"e2e tests run on nixer.io modified version of newman with enabled cookie support."